features/step_definitions/kata.rb

```diff
@@ -1,229 +1,103 @@
-  cb.test_project_name = project.name
-  file_path = "kata/example-fedora-kata.yaml"
-  step %Q(I run oc create over ERB test file: #{file_path})
-  raise "Example kata pod creation failed" unless @result[:success]
-  logger.info("Waiting for 'example-fedora-kata' pod to be RUNNING")
-  step %Q/a pod becomes ready with labels:/, table(%{
-    | app=example-fedora-kata-app |
-  })
-  logger.info("Checking for runtime engine match...")
-  # 1. check pod's spec to make sure the runtimeClassName is 'kata'
-  pod_runtime_class_name = pod('example-fedora-kata').runtime_class_name
-  if pod_runtime_class_name != 'kata'
-    raise "Pod's runtimeclass name #{pod_runtime_class_name} should be `kata`"
-  end
-
-  logger.info("Checking for running process `qemu` with the pod...")
-  # 2. check there's a process with the pod name `qemu`
+Given /^Catalog source #{QUOTED} exists in #{QUOTED} namespace$/ do |catalog_source_name, project_name|
   step %Q/I switch to cluster admin pseudo user/
-  node_cmd = "ps aux | grep qemu"
-  @result = node(pod.node_name).host.exec_admin(node_cmd)
-  raise "No qemu process detected inside pod node" unless @result[:response].include? 'qemu'
+  project(project_name)
+  raise "Failed to create catalog source" unless catalog_source("#{catalog_source_name}").exists?
+  logger.info("Catalog source #{catalog_source_name} exists")
 end
 
-Given /^there is a catalogsource for kata container$/ do
+When /^I install sandboxed-operator in #{QUOTED} namespace$/ do |kata_ns|
+  step %Q/I store master major version in the :master_version clipboard/
   step %Q/I switch to cluster admin pseudo user/
-  step %Q/I use the "openshift-marketplace" project/
-  step %Q|I obtain test data file "kata/catalogsource.yaml"|
-  user.cli_exec(:apply, f: 'catalogsource.yaml')
-  step %Q/a pod becomes ready with labels:/, table(%{
-    | olm.catalogSource=redhat-marketplace |
-  })
+  step %Q|I obtain test data file "kata/release-#{cb.master_version}/deployment.yaml"|
+  @result = user.cli_exec(:apply, f: "deployment.yaml")
+  raise "Failed to install sandboxed-operator" unless @result[:success]
 end
 
-Given /^the kata-operator is installed(?: to #{OPT_QUOTED})? using OLM(?: (CLI|GUI))?$/ do | kata_ns, install_method |
-  ensure_admin_tagged
-  kata_ns ||= "openshift-sandboxed-containers-operator"
-  kata_config_name = "example-kataconfig"
-  step %Q/I store master major version in the :master_version clipboard/
-  raise "Kata operator OLM installation only supported for OCP >= 4.8" unless cb.master_version >= "4.8"
-  install_method ||= 'CLI'
-  # first check pre-req
-  step %Q/I switch to cluster admin pseudo user/
-  project('openshift-marketplace')
-  unless catalog_source('qe-app-registry').exists?
-    logger.info("Kata installation depends on `qe-app-registry`, which is missing in this cluster, calling step to create it...")
-    step %Q/I create "qe-app-registry" catalogsource for testing/
+Then /^Operator should be up and running$/ do
```

Comment:
@valtair1891 same thing with the cap of the first word.   Also, I would be more specific as to what operator you are checking.  Just the word `operator` is too generic.

Reply:
@valtair1891 yes, please change it since the step_definitions are `global`.